web/admin/src/views/LocationsListPage.vue

```diff
@@ -2,20 +2,33 @@
 import { ref, onBeforeMount } from 'vue'
 import { useAuthUserStore } from '@/stores/authUser'
 import { collection, getFirestore, query, where, doc, getDocs, addDoc, updateDoc, orderBy } from 'firebase/firestore'
+import SortableTableHeader from '../components/SortableTableHeader.vue';
```

Comment:
```suggestion
import SortableTableHeader from '@/components/SortableTableHeader.vue'
```
You can use `@` to reference the root of the project, no need to use relative paths.  We have set this up in vite.config.js.

No need for semicolons at the end of the line.